src/ioctls/vm.rs

```diff
@@ -812,4 +855,25 @@ mod tests {
         let msi = kvm_msi::default();
         assert!(vm.signal_msi(msi).is_err());
     }
+
+    /// Providing the `kvm_enable_cap` structure filled with default() should
+    /// always result in a failure as it is not a valid capability.
+    #[test]
+    fn test_enable_cap_failure() {
+        let kvm = Kvm::new().unwrap();
+        let vm = kvm.create_vm().unwrap();
+        let cap: kvm_enable_cap = Default::default();
+        assert!(vm.enable_cap(&cap).is_err());
+    }
+
+    #[test]
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64",))]
+    fn test_enable_split_irqchip_cap() {
```

Comment:
Please add a comment and explain that cap.args[0] is the number of routes reserved for userspace IOAPICs or add a link to the full explanation: https://elixir.bootlin.com/linux/latest/source/Documentation/virtual/kvm/api.txt#L4606

It took me some time to understand what this does.